streamer/periodconcat_node.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Concatenates inputs into periods by creating a master DASH/HLS file."""
+
+import os
+import re
+from typing import List, Tuple
+from xml.etree import ElementTree
```

Comment:
Please add `xml.etree` to the list of dependencies in `docs/source/prerequisites.rst`, as well as the `install_requires` list in `setup.py`.

Reply:
`xml` is from the standard library.
https://docs.python.org/3/library/xml.etree.elementtree.html
we don't need to list it as a dependency right?

Reply:
I stand corrected!  No need to call it a dependency.